src/modules/localeq/Map.qml

```diff
@@ -101,7 +114,13 @@ Column {
                     marker.coordinate = coordinate
                     map.center.latitude = coordinate.latitude
                     map.center.longitude = coordinate.longitude
-                    console.log(coordinate.latitude, coordinate.longitude)
+                    
+                    geocodeModel.query = QtPositioning.coordinate(coordinate.latitude, coordinate.longitude)
```

Comment:
missed in commit message, this is a start of implementing ReverseGeocoding, console log shows output, but translated to degrees, not text yet